flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCInputFormatTest.java

```diff
@@ -101,6 +105,33 @@ public void testIncompleteConfiguration() throws IOException {
 	}
 
 	@Test
+	public void defaultFetchSizeIsUsedIfNotConfiguredOtherwise() throws SQLException {
+		jdbcInputFormat = JDBCInputFormat.buildJDBCInputFormat()
+			.setDrivername(DRIVER_CLASS)
+			.setDBUrl(DB_URL)
+			.setQuery(SELECT_ALL_BOOKS)
+			.setRowTypeInfo(ROW_TYPE_INFO)
+			.finish();
+		jdbcInputFormat.openInputFormat();
+		assertThat(jdbcInputFormat.getStatement().getFetchSize(), equalTo(1));
```

Comment:
That is equals 1 is an internal detail of the derby jdbc driver is it not? Not sure how useful this test is.